client/my-sites/checkout/checkout-thank-you/header.jsx

```diff
@@ -382,12 +407,24 @@ export class CheckoutThankYouHeader extends PureComponent {
 			hasFailedPurchases,
 			translate,
 			primaryPurchase,
+			purchases,
 			selectedSite,
 			displayMode,
 			isAtomic,
 		} = this.props;
 		const headerButtonClassName = 'button is-primary';
 		const isConciergePurchase = 'concierge' === displayMode;
+		const isSearch = purchases && purchases[ 0 ].productType === 'search';
+
+		if ( isSearch ) {
+			return (
+				<div className="checkout-thank-you__header-button">
+					<button className={ headerButtonClassName } onClick={ this.goToCustomizer }>
+						{ translate( 'Try Search and customize it now.' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **19** times:
`translate( 'Try Search and customize it now' )` *ES Score: 9*